support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -173,10 +180,12 @@ EvolutionSystem:
           Width: 38.415              # 2.5 * (x_B - x_A)
           Center: [0.0, 0.0, 0.0]
 
+# Half power was determined to filter only the last term up to N = 19
+# since we support up to 20 gridpoints.
```

Comment:
How did you determine this? I need a bit more information to understand this comment.

Reply:
@geoffrey4444 and I sat down and he showed me how to determine a good half power algebraically by solving for m in the exponential filtering documentation when i = 20, N = 19 and $\alpha$ = 36.0